src/app/(app)/attendance/components/CurrentAttendance.tsx

```diff
@@ -0,0 +1,193 @@
+"use client";
+import { Button } from "@/components/ui/button";
+import { Card, CardContent } from "@/components/ui/card";
+import { Switch } from "@/components/ui/switch";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+import { AttendanceStatus, MemberRole, Shift } from "@/constant/constant";
+import { User } from "@/types";
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import React, { useMemo, useState } from "react";
+
+export interface Attendance {
+  id: number;
+  name: string;
+  shift: Shift;
+  startTime: string;
+  endTime: string;
+  status: AttendanceStatus;
+}
+
+const attendance: Attendance[] = [
+  {
+    id: 1,
+    name: "John Doe",
+    shift: Shift.ShiftA,
+    startTime: "08:00",
+    endTime: "16:00",
+    status: AttendanceStatus.Approve,
+  },
+  {
+    id: 2,
+    name: "Jane Smith",
+    shift: Shift.ShiftA,
+    startTime: "16:00",
+    endTime: "00:00",
+    status: AttendanceStatus.Approve,
+  },
+  {
+    id: 3,
+    name: "Bob Johnson",
+    shift: Shift.ShiftA,
+    startTime: "00:00",
+    endTime: "08:00",
+    status: AttendanceStatus.Approve,
+  },
+];
+
+type Props = {
+  loginUser: User;
+};
+export const CurrentAttendance = ({ loginUser }: Props) => {
+  const [todayAttendance, setTodayAttendance] = useState(attendance);
+
+  const onChangeAttendanceStatus = (id: number) => {
```

Comment:
We use `on` prefix with props/callbacks only. Let's use handle

Reply:
Fixed